client/scripts/14araw.lua

```diff
@@ -137,11 +144,11 @@ function showdata(usbpacket)
 end
 
 
-
 function sendRaw(rawdata, options)
 	print(">> ", rawdata)
 	
 	local flags = lib14a.ISO14A_COMMAND.ISO14A_NO_DISCONNECT + lib14a.ISO14A_COMMAND.ISO14A_RAW
+	if topaz_mode == true then flags = flags + lib14a.ISO14A_COMMAND.ISO14A_TOPAZMODE end
```

Comment:
the variable topaz_mode is not global.  You need to call it like  options is done.

Reply:
You might have guessed already: I don't speak LUA. 

So it should be `'corcpx:t3'` ? And btw: why do we have `c` two times?

So it should be `options.topaz_mode` instead?

Reply:
The double c is redundant aswell.  One is enough.

The options param, needs to be called with topaz_mode ( https://github.com/iceman1001/proxmark3/blob/master/client/scripts/14araw.lua#L111 )

and use/checked like you wrote.

Reply:
looking at the getopt,  I'm guessing it should be a 'd'  in order to enable debug mode, instead of an extra 'c'